website/data/docs-remote-plugins.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I've left an empty array here for now. Entries for remote plugins would be something like:

```
[
  {
    title: "Docker",
    path: "docker",
    repo: "hashicorp/packer-plugin-docker",
  }
]
```

Entries can optionally also contain `branch` and `artifactDir` values, these default to `main` and `.docs-artifacts` respectively.

Reply:
Maybe add a commented out example of the format here just for the future?

Reply:
It's only JSON5 that has comments though, right? Will definitely add some docs to the README on this 👍